cmds/rm/rm.go

```diff
@@ -69,10 +64,10 @@ func rm(files []string) error {
 			return err
```

Comment:
it removes them

rminnich-macbookpro2:bb rminnich$ touch /tmp/{b,c}
rminnich-macbookpro2:bb rminnich$ ls -l !$
ls -l /tmp/{b,c}
-rw-r--r--  1 rminnich  wheel  0 Aug  8 08:45 /tmp/b
-rw-r--r--  1 rminnich  wheel  0 Aug  8 08:45 /tmp/c
rminnich-macbookpro2:bb rminnich$ rm /tmp/a /tmp/b /tmp/c
rm: /tmp/a: No such file or directory
rminnich-macbookpro2:bb rminnich$ ls -l /tmp/{b,c}
ls: /tmp/b: No such file or directory
ls: /tmp/c: No such file or directory
rminnich-macbookpro2:bb rminnich$